modules/internal/ChapelBase.chpl

```diff
@@ -1496,8 +1496,11 @@ module ChapelBase {
   inline proc >(a: uint(64), param b: uint(64)) {
     return __primitive(">", a, b);
   }
+  inline proc >(param a: uint(?w), b: uint(w)) param where a == 0 {
```

Comment:
I did try changing to uint(64)s on these overloads and, as feared, my new test didn't pass -- I believe because comparisons on smaller uint types are caught by some of the smaller int overloads first.